src/smc-project/jupyter/package.json

```diff
@@ -14,8 +14,8 @@
   "dependencies": {
     "enchannel-zmq-backend": "^2.6.1",
```

Comment:
ok, thank you, and a consistent monorepo sounds good (to eliminate the combinatorial explosion)